README.md

```diff
@@ -82,6 +82,18 @@ import '@constructor-io/constructorio-ui-quizzes/styles.css';
 
 ### Known Issues
 
+**Javascript version**
+
+The default exported javascript syntax is ESNext, the latest supported syntax of JavaScript. 
```

Comment:
What do you think about this?

This library exports two different versions
* CommonJS (cjs)
* ECMAScript Modules (mjs)

If you are using an `import` statement, `mjs` version will be imported.
If you are using a `require` statement, `cjs` version will be imported.

Since `mjs` version is compiled to `ESNext`, there could be features that are not supported by older Chrome versions (ex. optional chaining). If you are receiving an error similar to the following error, please import the `cjs` version which is compiled to ES2015.

`import CioQuiz from '@constructor-io/constructorio-ui-quizzes/commonJs'`

Reply:
> If you are using an import statement, mjs version will be imported.
If you are using a require statement, cjs version will be imported.

Settled on Slack